editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { Component } from 'element';
+import PanelBody from 'components/panel-body';
+import PanelBodyToggle from 'components/panel-body-toggle';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+
+class PostStatus extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			opened: true,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
+		this.setState( {
+			opened: ! this.state.opened,
+		} );
+	}
+
+	render() {
+		return (
+			<PanelBody>
+				<PanelBodyToggle
```

Comment:
(I'm unsure:) Do you think it'd make any more sense to render this as a component which accepts children and manages "open" state internally?

```
<PanelToggleSection initialOpen={ true }>
	<div className="editor-sidebar-post-status__row"></div>
</PanelToggleSection>
```